mozilla/kuma
# Impelementing Sean Martell's suggested homepage fixes.  Provide gradient header to demo, zone, and homepages.

This one is big but awesome.

This PR includes all of Sean's suggested homepage updates, all being minor.  A few font fixes here and there, color changes, etc.  http://cl.ly/image/2Q2y0W280f2k/o

Moreover, this simplifies zone styles for writers.  Instead of setting a zone color on like 3 selectors as they do now, they'll simply need to set a homepage background color.  

_ONCE THIS IS MERGED, WE NEED TO GO TO EVERY ZONE IN PRODUCTION TO CHANGE THE SELECTOR WHICH CHANGES THE ZONE COLOR_  Before merging, someone should make sure I'm around so I can change zone colors right after we push.

How to spot-check:
1.  Look at a basic wiki page -- it should look good, with a white background for the content.
2.  Change a document zone's CSS so that the `body` has the zone `background-color`, not the selector we initially used for this (`.zone .zone-landing-header, .zone #main-header, .zone .zone-article-header`).  Look at the zone landing page -- does it look correct?  Look at the zone subpage -- does it look correct?
3.  Check out the demos section -- does that look correct?

Let me know if you have questions.
